drivers/bluetooth/hci/Kconfig

```diff
@@ -68,9 +68,11 @@ config BT_NO_DRIVER
 
 endchoice
 
+DT_CHOSEN_Z_BT_UART := zephyr,bt-uart
```

Comment:
A comment would be nice for these. I'd assume the variable is set because it's used in many places, not just to hack around a comma.

Could have a

    comma := ,

definition somewhere instead maybe, and use that.

Reply:
Up to you. Could add a short comment like "# Workaround for comma in value" instead.

Reply:
Still missing a comment.

I'm about to push a Kconfiglib update that exposes the parsing location as `kconf.filename/linenr` btw, for use in preprocessor functions.